temporalcli/commandsmd/commands.md

```diff
@@ -1519,17 +1519,15 @@ This command provides the following task queue statistics:
 - `ApproximateBacklogCount`: The approximate number of tasks backlogged in this
   task queue.  May count expired tasks but eventually converges to the right
   value.
-- `ApproximateBacklogAge`: How far "behind" the task queue is running. This is
-  the difference in age between the oldest and newest tasks in the backlog,
-  measured in seconds.
+- `ApproximateBacklogAge`: Approximate age of the oldest task in the backlog, based on its creation time, measured in seconds.
 - `TasksAddRate`: Approximate rate at which tasks are being added to the task
   queue, measured in tasks per second, averaged over the last 30 seconds.
   Includes tasks dispatched immediately without going to the backlog
-  (sync-matched tasks), as well as tasks added to the backlog.
+  (sync-matched tasks), as well as tasks added to the backlog. However, not all tasks dispatched to the workers are counted in this metric. For more information please see [docs](https://github.com/temporalio/api/blob/b4bdd8035cd1883aa96cbad5bb0e582850feea5f/temporal/api/taskqueue/v1/message.proto#L116).
```

Comment:
plan is to replace the hyper link with temporal docs link - haven't done so yet since docs changes have not been published :(  (*yet*)

Reply:
If the docs page isn't available yet, we'll have to write this without the link and then update it later (after the release). See what you think about the wording in my PR (sorry ours crossed each other in flight!): https://github.com/temporalio/cli/pull/669/files#diff-b4c85d71e771b938a0779d1e4113e068c19dd896aac86c04fe1828dd5a15c63f